packages/@react-spectrum/card/src/WaterfallLayout.tsx

```diff
@@ -113,8 +105,8 @@ export class WaterfallLayout<T> extends BaseLayout<T> implements KeyboardDelegat
       } else if (node.props.width && node.props.height) {
         let nodeWidth = node.props.width;
         let nodeHeight = node.props.height;
-        let scaledHeight = Math.round(nodeWidth * ((itemWidth) / nodeHeight));
-        height = Math.max(this.minItemSize.height, Math.min(this.maxItemSize.height, scaledHeight)) + this.itemPadding;
+        let scaledHeight = Math.round(nodeHeight * ((itemWidth) / nodeWidth));
```

Comment:
Found this when digging into the max height for waterfall layout, had the `nodeHeight` and `nodeWidth` flipped when calculating the scaled height. Doesn't really affect the end result since WaterfallLayout uses the Card's intrinsic height/width as the final size but was confusing me when I was poking around haha